src/widgets/current-refinements/__tests__/current-refinements-test.ts

```diff
@@ -389,39 +417,45 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/current-ref
         );
         widget.render!(
           createRenderOptions({
-            results: new SearchResults(helper.state, [
-              createSingleSearchResponse({
-                facets: [
-                  {
-                    name: 'facet',
-                    exhaustive: true,
-                    data: {
-                      'facet-val1': 1,
-                      'facet-val2': 2,
-                    },
-                  },
-                  {
-                    name: 'extraFacet',
-                    exhaustive: true,
-                    data: {
-                      'extraFacet-val1': 42,
-                      'extraFacet-val2': 42,
-                    },
-                  },
-                ],
-                // @ts-ignore wrong types for `disjunctiveFacets`
-                disjunctiveFacets: [
-                  {
-                    name: 'disjunctiveFacet',
-                    exhaustive: true,
-                    data: {
-                      'disjunctiveFacet-val1': 3,
-                      'disjunctiveFacet-val2': 4,
-                    },
-                  },
-                ],
-              }),
-            ]),
+            scopedResults: [
+              {
+                indexId: 'index_name',
+                results: new SearchResults(helper.state, [
+                  createSingleSearchResponse({
+                    facets: [
+                      {
+                        name: 'facet',
+                        exhaustive: true,
+                        data: {
+                          'facet-val1': 1,
+                          'facet-val2': 2,
+                        },
+                      },
+                      {
+                        name: 'extraFacet',
+                        exhaustive: true,
+                        data: {
+                          'extraFacet-val1': 42,
+                          'extraFacet-val2': 42,
+                        },
+                      },
+                    ],
+                    // @ts-ignore @TODO: missing types for facets in the response
+                    disjunctiveFacets: [
```

Comment:
same remark here and in the other instances, we need to make sure we don't ignore these errors, since they actually behave wrong (skip helper steps, which won't happen IRL)